app/theme/default_100_percent/common/avatars/avatarlist

```diff
@@ -0,0 +1,15 @@
+avatar_braveorange.png
+avatar_orangepeel.png
+avatar_nana.png
+avatar_envy.png
+avatar_clearskies.png
+avatar_bondyblue.png
+avatar_blurple.png
+avatar_darknight.png
+avatar_orangepeel_shape.png
+avatar_nana_shape.png
+avatar_envy_shape.png
+avatar_clearskies_shape.png
+avatar_bondyblue_shape.png
+avatar_blurple_shape.png
+avatar_darknight_shape.png
```

Comment:
This file exists for code generation of avatars in the correct order, in the event we need to re-number resource IDs